frontend/src/upload.js

```diff
@@ -345,6 +358,9 @@ $(document).ready(function() {
       class: 'icon-copy',
       'data-l10n-id': 'copyUrlHover'
     });
+    if (!allowedCopy()) {
+      $copyIcon.addClass('disabled');
```

Comment:
I'll push changes you asked, but for this change `#copy-btn:disabled` rule cannot be reused because `.icon-copy` is an image but I used `disabled` attribute instead of `.disabled`